zanata-war/src/main/java/org/zanata/action/ValidateEmailAction.java

```diff
@@ -60,6 +61,11 @@
     @In
     EmailChangeService emailChangeService;
 
+    @Create
+    public void onCreate() {
+        identity.checkLoggedIn();
```

Comment:
@carlosmunoz Can you think of some way (in Seam, or in CDI) of intercepting bean creation, controlled by a class-level annotation like `@CheckLoggedIn`?  The idea here is to prevent the JSF page from loading with an authorisation exception, similar to what we used to do in pages.xml.  The only thing I can think of is a CDI extension, but I'm not sure how much work it would be.